pkg/util/util.go

```diff
@@ -83,6 +102,8 @@ func Merge(commonSpecs []v1.JaegerCommonSpec) *v1.JaegerCommonSpec {
 			affinity = commonSpec.Affinity
 		}
 
+		mergeDNSPolicyAndHostNetwork(&dnsPolicy, &commonSpec.DNSPolicy, &hostNetwork, &commonSpec.HostNetwork)
```

Comment:
I would prefer the code to be in this function, like the other merge routines. The merge function isn't that complex after all.

Reply:
I wrote it like this for code limit checks. It is too complex for exceeding 50 lines.

Reply:
Don't worry about codeclimate. Every time it complains, I check there and will discard its review if it doesn't make sense.

Reply:
fine